src/pages/en/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,405 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Upgrade Guide
+description: How to upgrade your project to the latest version of Astro.
+i18nReady: false
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+
+This guide will help you migrate from Astro v1 to Astro v2.
+
+Need to upgrade an older project to v1? See our [older migration guide](/en/guides/upgrade-to/v1/).
+
+## Upgrade Astro
+
+Update your project's version of Astro to the latest version using your package manager. If you're using Astro integrations, please also update those to the latest version.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # Upgrade to Astro v2.x
+  npm install astro@latest
+  
+  # Example: upgrade React and Tailwind integrations
+  npm install @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # Upgrade to Astro v2.x
+  pnpm install astro@latest
+
+  # Example: upgrade React and Tailwind integrations
+  pnpm install @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # Upgrade to Astro v2.x
+  yarn add astro@latest
+  
+  # Example: upgrade React and Tailwind integrations
+  yarn add @astrojs/react@latest @astrojs/tailwind@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+## Astro 2.0 Breaking Changes
+
+Astro v2.0 includes some breaking changes, as well as the removal of some previously deprecated features. If your project doesn't work as expected after upgrading to v2.0, check this guide for an overview of all breaking changes and instructions on how to update your codebase.
+
+See [the changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) for full release notes.
+
+### Removed: Support for Node 14
+
+Node 14 is scheduled to reach its End of Life in April 2023. 
+
+Astro 2.0 drops Node 14 support entirely, so that all Astro users can take advantage of Node's more modern features.
+
+#### What should I do?
+
+ Check that both your development environment and your deployment environment are using **Node.js `16.12.0` or later**.
+
+1. Check your local version of node using:
```

Comment:
Generally when we write "node" in lowercase we are referring to the terminal program, so maybe we should mark it as code?

```suggestion
1. Check your local version of `node` using:
```

or just uppercase it:

```suggestion
1. Check your local version of Node using:
```